src/storage/txn/actions/commit.rs

```diff
@@ -320,4 +321,41 @@ pub mod tests {
         must_err(&mut engine, k, ts(60, 0), ts(65, 0));
         must_succeed(&mut engine, k, ts(60, 0), ts(80, 0));
     }
+
+    #[test]
+    fn test_inherit_last_change_info_from_lock() {
+        use pd_client::FeatureGate;
+
+        use crate::storage::txn::sched_pool::set_tls_feature_gate;
+
+        let feature_gate = FeatureGate::default();
+        feature_gate.set_version("6.5.0").unwrap();
```

Comment:
You have removed the `set version to 6.5.0` in case `test_mess()`, why does this case need to be set 6.5.0? TiKV version hasn't reached 6.5 currently?

Reply:
In https://github.com/tikv/tikv/pull/13721, I removed the checks about `last_change_ts` in `test_mess`. So, it doesn't depend on this feature now.

When the PR was created, the bumping version PR was not merged yet.